src/lib/db/seed.ts

```diff
@@ -133,6 +140,37 @@ const entitiesContents: Record<string, string[]> = {
 	],
 };
 
+const generateTodo = async (db: Database, persona_id: number, space_id: number) => {
+	const list = await db.repos.entity.create(
+		persona_id,
+		{
+			type: 'Collection',
+			name: 'Grocery List',
+		},
+		space_id,
+	);
+	if (!list.ok) {
+		log.warn('Issue generating todo list');
+		return;
+	}
+	const items = ['eggs', 'milk', 'bread'];
+	for (const item of items) {
+		const result = await db.repos.entity.create(
+			persona_id,
+			{
+				type: 'Note',
+				content: item,
+			},
+			space_id,
+		);
+		if (!result.ok) {
+			log.warn('Issue generating todo item');
+			return;
+		}
+		await db.repos.tie.create(list.value.entity_id, result.value.entity_id, 'HasItem');
```

Comment:
Had some overlapping suggestions here so it's coming as a big block.

- Instead of warning and returning early in 2 places, it's more succinct to use `unwrap`, and I think we'd prefer to have its behavior of throwing an error if any part of seeding fails.
- Use `unwrap` on `db.repos.tie.create` even though the return value isn't used so any errors will make seeding fail.

```suggestion
	const list = unwrap(
		await db.repos.entity.create(
			persona_id,
			{
				type: 'Collection',
				name: 'Grocery List',
			},
			space_id,
		),
	);
	const itemsContent = ['eggs', 'milk', 'bread'];
	for (const content of itemsContent) {
		const item = unwrap(
			await db.repos.entity.create(
				persona_id,
				{
					type: 'Note',
					content,
				},
				space_id,
			),
		);
		unwrap(await db.repos.tie.create(list.entity_id, item.entity_id, 'HasItem'));
```